src/AerospikeCache.php

```diff
@@ -0,0 +1,133 @@
+<?php
+declare(strict_types=1);
+
+namespace Lmc\AerospikeCache;
+
+use Aerospike;
```

Comment:
This is from `global` namespace and it should not be here. (same on #14)

Reply:
Removed